pkgs/development/libraries/umockdev/default.nix

```diff
@@ -59,10 +59,13 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeCheckInputs = [
     python3
-    which
+    systemdMinimal
```

Comment:
Maybe we should patch it in https://github.com/martinpitt/umockdev/blob/6a1d57a80a511c8a6cc8a0756673d4f5f0f53146/src/umockdev-record.vala#L226 so that it works at runtime?

Reply:
Ah, I completely forgot that when looking at https://github.com/NixOS/nixpkgs/issues/280697. The way it is now, `udevadm` isn't necessarily in the program's path and even if it is, it might not be from the same systemd version it has in its inputs. Patching that is kinda the only clean way, right?

Reply:
We either require everything using umockdev to provide a udevadm binary in the environment umockdev runs in, or we patch in some path to it.

Considering it's only used for testcases, either way is probably fine?

Reply:
I guess most of the packages tested with the help of umockdev already depend on systemd. But patching it here explicitly might be nicer in case there is an exception to this.

Reply:
I patched it in, and could then remove explicitly referencing to it from `nativeCheckInputs`. PTAL.